catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/LoopBrickTest.java

```diff
@@ -0,0 +1,156 @@
+ */
+
+package org.catrobat.catroid.uiespresso.content.brick;
+
+import android.support.test.espresso.action.CoordinatesProvider;
+import android.support.test.espresso.action.GeneralLocation;
+import android.support.test.espresso.action.GeneralSwipeAction;
+import android.support.test.espresso.action.Press;
+import android.support.test.espresso.action.Swipe;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.bricks.ChangeYByNBrick;
+import org.catrobat.catroid.content.bricks.LoopEndBrick;
+import org.catrobat.catroid.content.bricks.RepeatBrick;
+import org.catrobat.catroid.formulaeditor.InterpretationException;
+import org.catrobat.catroid.ui.ScriptActivity;
+import org.catrobat.catroid.uiespresso.util.BaseActivityInstrumentationRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.longClick;
+
+import static android.support.test.espresso.assertion.ViewAssertions.doesNotExist;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.checkIfBrickAtPositionShowsString;
+import static org.catrobat.catroid.uiespresso.content.brick.BrickTestUtils.onScriptList;
+
+@RunWith(AndroidJUnit4.class)
+public class LoopBrickTest {
+	private int brickPosition;
+	private Script script;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		brickPosition = 1;
+		int timesToRepeat = 3;
+		int changeYMovement = -10;
+
+		LoopEndBrick endBrick = new LoopEndBrick();
+		RepeatBrick repeatBrick = new RepeatBrick(timesToRepeat);
+		repeatBrick.setLoopEndBrick(endBrick);
+
+		script = BrickTestUtils.createProjectAndGetStartScript("LoopBrickTest1");
+		script.addBrick(repeatBrick);
+		script.addBrick(new ChangeYByNBrick(changeYMovement));
+		script.addBrick(endBrick);
+
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Test
+	public void repeatBrickTest() throws InterpretationException {
+
+		checkSetUpBrickArrangement();
+
+		deleteBrickAtPosition(brickPosition);
+
+		checkIfBrickAtPositionShowsString(1, R.string.brick_change_y_by);
+		onView(withText(R.string.brick_loop_end))
+				.check(doesNotExist());
+
+		addLoopAtPosition(R.string.brick_repeat, brickPosition);
+
+		// selects block and drags it from Top_Center (of the block) to the bottom of the device screen
+		onScriptList().atPosition(2)
```

Comment:
please put into a function and name it something like dragBrickAtPositionToBottom